2section_getting_started.rb

```diff
@@ -0,0 +1,85 @@
+=begin
+    This code was made for practicing ruby :D
+    Author: @roa-sh
+=end
+
+5.times{puts "Hello world for everybody"}
+puts
+
+#-----------------The puts method---------
+#and can prints aritmetic operations too
```

Comment:
for comments put a space after the # (https://github.com/rubocop/ruby-style-guide#hash-space)